components/menu/demo/menu.html

```diff
@@ -20,6 +20,35 @@
 
 		<d2l-demo-page page-title="d2l-menu">
 
+			<h2>Dark Theme</h2>
```

Comment:
Did you mean to commit this demo? Just wondering based on the PR description

Reply:
I did not! Will remove thx.